internal/pkg/log/log.go

```diff
@@ -2,105 +2,123 @@
 package log
 
 import (
-	"io"
-	"os"
+	"context"
```

Comment:
This package needs more test coverage. We should have parameterized tests, where we test every logging method.

Reply:
Makes sense. Most of the tests right now are in the `zap` package, but I'll add some to cover the interface functions as well.